scripts/g.extension/g.extension.py

```diff
@@ -2200,6 +2200,20 @@ def resolve_source_code(url=None, name=None, branch=None):
                 url_validated = True
             except:
                 pass
+            # Test if github repo exists (need to use API)
+            try:
+                open_url = urlopen('https://api.github.com/repos/{}/{}'.format(*url.split("/")[-2:]))
+                open_url.close()
+                url_validated = True
+            except:
```

Comment:
It will be good catch 'GitHub API rate limit exceeded.' exception and inform user. (I'm going to add it in next commit).

Reply:
Thanks for adding that!